app/templates/ui/app/user/user.service.spec.js

```diff
@@ -13,7 +13,7 @@
     };
 
     beforeEach(function() {
-      bard.appModule('app.user');
+      bard.appModule('app.user', 'app');
```

Comment:
Whatever is in app module, or among app module dependencies that is required here, we need to make available to app.user module without adding it here..